### PR TITLE
Update deprecated tag and fix broken tags

### PR DESCRIPTION
**Pull request by tygyh**
## Description

Fix mostly silent errors in HTML files. These would become problem if they weren't the last or empty statements in their scopes.

### CODE REVIEW

File: resources/server/api/ogc/templates/wfs3/describeCollection.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <div class="row">
     <div class="col-md-6">
         <h1><a title="View items of: {{ title }}" href="{% for link in links %}
-        {% if link.type == "text/html" and link.rel == "items" %}
+        {% if link.type == &quot;text&sol;html&quot; and link.rel == &quot;items&quot; %}
```
Comment (nyalldawson) [inline resources/server/api/ogc/templates/wfs3/describeCollection.html:7]: I don't think these changes are correct -- were they deliberate?

File: resources/server/api/ogc/templates/wfs3/header.html [hunk 2]
```diff
@@ -13,12 +13,12 @@
     <link rel="stylesheet" href="https://unpkg.com/leaflet.fullscreen@3.0.2/Control.FullScreen.css"
             integrity="sha384-PALxxXbG28eiE9bT7yKmsbjd4Ta3qKMLlzsrp6G+05f4Nltt4ktlg1l7e4RPeNEa"
             crossorigin=""/>
-    <link rel="stylesheet" href="{{ static( "style.css" ) }}" />
+    <link rel="stylesheet" href="{{ static( &quot;style.css&quot; ) }}" />
```
Comment (nyalldawson) [inline resources/server/api/ogc/templates/wfs3/header.html:16]: Same here, I think these are incorrect

Comment (tygyh) [reply resources/server/api/ogc/templates/wfs3/header.html:16]: Yes, `{{ static( ` and ` ) }}` become strings but the parts between does not. I assume my fix is wrong, but I don't know it should be written instead

File: resources/server/api/ogc/templates/wfs3/describeCollection.html (hunk 1 above)
Comment (tygyh) [reply resources/server/api/ogc/templates/wfs3/describeCollection.html:7]: Yes, `{{ static( ` and ` ) }}` become strings but the parts between does not. I assume my fix is wrong, but I don't know it should be written instead

File: resources/server/api/ogc/templates/wfs3/header.html (hunk 2 above)
Comment (tygyh) [reply resources/server/api/ogc/templates/wfs3/header.html:16]: Yes, {{ static(  and  ) }} become strings but the parts between does not. I assume my fix is wrong, but I don't know how it should be written instead

**Pull request closed without merging**